### ISSUE TITLE
Question - Detecting Keypresses without Interfering

### ISSUE THREAD

**Issue by latenitefilms** [Hammerspoon/hammerspoon#1017]
If I use **hs.hotkey.bind** to trigger an action based on a keypress, then it will override whatever the original keys were supposed to do. For example, if I use the "i" key, then I won't be able to type normally - as Hammerspoon will basically "take over" what the "i" key does now.

I think it might be possible to use **hs.eventtap** to detect key presses without stopping whatever their original function was - is that correct? I've tried looking through the documentation, but I haven't been able to work it out yet. Does any one have any examples of how I could achieve this?

Basically, I want to come up with a way that the shortcuts keys within an application work as normal - EXCEPT Hammerspoon "sees" the keypresses, so that I can trigger other actions on top of what the application does natively. Any ideas?

Thanks in advance!

**Comment by latenitefilms** [comment]
Never mind, finally worked it out!

```
local eventtap = hs.eventtap.new({ hs.eventtap.event.types.keyDown }, function(event)
  whichKey = event:getKeyCode()
  whichFlags = event:getFlags()
  if (hs.keycodes.map[whichKey] == "c") and (whichFlags['cmd']) then
  	print("CMD+C PRESSED")
  end
  return false
end)
eventtap:start()
```
